sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -57,7 +57,13 @@ case class LocalRelation(output: Seq[Attribute], data: Seq[InternalRow] = Nil)
     LocalRelation(output.map(_.newInstance()), data).asInstanceOf[this.type]
   }
 
-  override protected def stringArgs = Iterator(output)
+  override protected def stringArgs: Iterator[Any] = {
+    if (data.isEmpty) {
+      Iterator("Empty", output)
```

Comment:
Just to make sure, do you prefer the following output?
```
LocalTableScan EmptyRelation, [_1#58,_2#59,_3#60]
```

Instead of 
```
LocalTableScan Empty, [_1#58,_2#59,_3#60]
```